client.py

```diff
@@ -13,15 +13,29 @@
 my_user = structure.User_obj('Funing','456',chats=[]) # need to modify after demo, maybe change to a string
 my_chat_room = ''
 
-def add_user_client():
+def user_login_client():
```

Comment:
My thought process was – the client sends the username,password to the server. The client will be sent an ACK (user w/ user,password combo does exist), and the loop will be exited, or a NAK (user w/o user,password combo doesn't exist). Thoughts? Wasn't sure how to have the server request the username/password @ start.